vignettes/create-individual.Rmd

```diff
@@ -55,3 +55,30 @@ setParameterValuesByPath(parameterPaths = individual$distributedParameters$paths
                          values = individual$distributedParameters$values,
                          simulation = sim)
 ```
+
+## Adding enzyme ontogenies
+The protein expression database of PK-Sim includes ontogeny information for some proteins (see (OSPS manual)[https://docs.open-systems-pharmacology.org/working-with-pk-sim/pk-sim-documentation/pk-sim-expression-data#settings-in-the-protein-expression-tab]). For a protein molecule present in the simulation, it is possible to add the ontogeny information on one of the pre-defined proteins. For example, it is possible to set the ontogeny of the protein `MyProtein` to the value of the ontogeny of a CYP3A4 enzyme for the specified individual. The list of supported ontogenies is stored in the `StandardOntogeny`-list.
+
+```{r addOntogeny}
+library(ospsuite)
+
+# All supported ontogenies
+print(StandardOntogeny)
+
+#Create the ontogeny for the protein "MyProtein" based on ontology of CYP3A4
+myProteinOntogeny <- MoleculeOntogeny$new(molecule = "MyProtein", ontogeny = StandardOntogeny$CYP3A4)
+print(myProteinOntogeny)
+
+# Add this ontogeny to the individual characteristics used to create the individual parameters set
+individualCharacterstics <- createIndividualCharacteristics(species = Species$Human,
+                                                            population = HumanPopulation$Japanese_Population,
+                                                            gender = Gender$Female,
+                                                            weight = 75,
+                                                            height = 1.75,
+                                                            heightUnit = "m",
+                                                            age = 43,
+                                                            moleculeOntogenies = myProteinOntogeny)
```

Comment:
I don't know if it's stated elsewhere that [kg] and [year(s]] are default units and that's why weightUnit and AgeUnit are omitted in this example.

Reply:
Added a comment on that.